qa/tasks/mgr/dashboard/helper.py

```diff
@@ -78,7 +78,11 @@ def _request(self, url, method, data=None):
         log.info("request %s to %s", method, url)
         if method == 'GET':
             self._resp = self._session.get(url)
-            return self._resp.json()
+            try:
+                return self._resp.json()
+            except ValueError as ex:
+                log.error("response: %s", self._resp.text)
+                log.exception("Failed to decode response: %s", str(ex))
```

Comment:
You don't need to add `str(ex)` to log.exception. it does that automatically. I'd use 
```python
log.exception("Failed to decode response: %s", self._resp.text)
```
instead.

**Also**, you're silently returning `None` here, Can we re-throw the exception here?

Reply:
fixed as you suggested